### PR TITLE
Request example for Create user on PHP: setDateOfBirth method is not correct

### PR DESCRIPTION
**Pull request by dyurchenko-epam**
https://jira.site1.hyperwallet.local/browse/HW-54154

### CODE REVIEW

File: src/Hyperwallet/Model/BankAccount.php [hunk 1]
```diff
@@ -784,11 +784,13 @@ public function getDateOfBirth() {
...
-     * @param \DateTime|null $dateOfBirth
+     * @param string $dateOfBirth |null
      * @return BankAccount
      */
-    public function setDateOfBirth(\DateTime $dateOfBirth = null) {
-        $this->dateOfBirth = $dateOfBirth == null ? null : $dateOfBirth->format('Y-m-d');
+    public function setDateOfBirth($dateOfBirth = null)
+    {
+        $stringToDate = $dateOfBirth === null ? null : new \DateTime($dateOfBirth);
+        $this->dateOfBirth = $stringToDate == null ? null : $stringToDate->format('Y-m-d');
```
Comment (aseveryn-hw) [inline src/Hyperwallet/Model/BankAccount.php]: As this date manipulation is done in several places, I suggest to make a common util method and use it in different places

Comment (dyurchenko-epam) [reply src/Hyperwallet/Model/BankAccount.php]: Done, thank you.

File: src/Hyperwallet/Util/StringToDataConverter.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+namespace Hyperwallet\Util;
+
+
+class StringToDataConverter
+{
+    static function convertStringToDate($string, $format)
```
Comment (aseveryn-hw) [inline src/Hyperwallet/Util/StringToDataConverter.php]: Can we cover this method with tests?

Comment (dyurchenko-epam) [reply src/Hyperwallet/Util/StringToDataConverter.php]: Done.

**Comment by dyurchenko-epam** [comment]
It's not necessary anymore because there is an another pr for solving this problem https://github.com/hyperwallet/program-portal/pull/215

**Pull request closed without merging**